src/concourse/pipelines/infrastructure/grafana_cloud/grafana_cloud_pipeline.py

```diff
@@ -0,0 +1,173 @@
+from concourse.lib.constants import REGISTRY_IMAGE
```

Comment:
I think this is probably an example of a pipeline that is more complicated via python than just doing it in yaml. It was a good exercise but it didn't really lend itself to this framework well. Granted there may be opportunities to optimize this that I didn't notice/take.